tests/scripts/thread-cert/Cert_5_3_02_RealmLocal.py

```diff
@@ -96,14 +96,14 @@ def test(self):
                 self.assertTrue(self.nodes[LEADER].ping(addr, size=256))
                 self.assertTrue(self.nodes[LEADER].ping(addr))
 
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1', size=256))
-        self.assertTrue(self.nodes[LEADER].ping('ff03::1'))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2, size=256))
+        self.assertTrue(self.nodes[LEADER].ping('ff03::1', num_responses=2))
```

Comment:
Oh, ok. I forgot there was a special Realm-Local All Thread Nodes address that are sent to SEDs. I thought it was just the Realm-Local All Nodes address. Then I'm sorry for making you make additional changes, but I think your first change was the best.